packages/myst-cli/src/build/tex/single.ts

```diff
@@ -52,6 +52,7 @@ export function mdastToTex(
     bibliography: templateYml?.style?.bibliography,
     printGlossaries,
     references,
+    ...frontmatter.settings?.myst_to_tex,
```

Comment:
We need to be a little more careful with this - currently we only set `printGlossaries` to `true` in one place: https://github.com/executablebooks/mystmd/blob/main/packages/myst-cli/src/build/tex/single.ts#L214 - a bunch of other places it must be false so we don't re-print a ton.

I think that line is where we want to consume `frontmatter.settings.myst_to_tex.printGlossaries` - everywhere else it still needs to be false.

But then we still need to unpack the other settings here...

Reply:
Hm, ok. I think I am just going to take it out for now.